packages/appstore/src/components/modals/wallet-success-dialog/wallet-success-dialog.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Button, Modal, Text, WalletCard } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { useActiveWallet } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { formatMoney } from '@deriv/shared';
+import { getAccountName } from 'Constants/utils';
+import './wallet-success-dialog.scss';
+
+const WalletSuccessDialog = observer(() => {
+    const { ui, traders_hub } = useStore();
+    const { setWalletModalActiveTab } = traders_hub;
+    const { is_wallet_creation_success_modal_open, toggleIsWalletCreationSuccessModalOpen } = ui;
+    const active_wallet = useActiveWallet();
+
+    return (
+        <Modal
+            className='wallet-success-dialog'
+            is_open={is_wallet_creation_success_modal_open}
+            toggleModal={toggleIsWalletCreationSuccessModalOpen}
+            has_close_icon={false}
+            small
+        >
+            <Modal.Body>
+                <div className='wallet-success-dialog__icon'>
+                    <WalletCard
+                        wallet={{
+                            currency: active_wallet?.currency_config?.display_code,
+                            icon: active_wallet?.icon,
+                            icon_type: active_wallet?.currency_config?.type,
+                            name: getAccountName({
+                                account_type: 'wallet',
+                                display_currency_code: active_wallet?.currency_config?.display_code,
+                            }),
+                            balance: formatMoney(active_wallet?.currency, active_wallet?.balance, true),
+                            jurisdiction_title: active_wallet?.landing_company_name,
+                            gradient_class: active_wallet?.gradient_card_class,
+                        }}
```

Comment:
just personal preference I think this one can be declare at the top as `const wallet_details` and  pass all the props there so it can be reference at `wallet={wallet_details}` to make it look abit cleaner.